integrationtest/org/daisy/dotify/formatter/test/resource-files/external-reference3-input.obfl

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<obfl xmlns="http://www.daisy.org/ns/2011/obfl" version="2011-1" xml:lang="und" hyphenate="false" xmlns:example="http://example.com">
+   <meta xmlns:dc="http://purl.org/dc/elements/1.1/">
+      <dc:title>Tests external-reference element</dc:title>
+      <dc:description>Tests that when a external-reference is followed by text without white space
+      in between, it is attached to the row that contains that text, not the row
+      before.</dc:description>
+   </meta>
+   <layout-master name="master" page-width="10" page-height="5" duplex="true">
+      <default-template>
+         <header/>
+         <footer/>
+      </default-template>
+   </layout-master>
+   <sequence master="master">
+      <block>
+        ⠁⠁⠁⠁⠁⠁⠁⠁⠁⠁
```

Comment:
To be sure that this tests the right thing, should be less than 10 characters. I had put 7 (see also corresponding PEF).